languages/elixir/exercises/concept/binary-matching/.docs/introduction.md

```diff
@@ -0,0 +1,52 @@
+Elixir provides an elegant syntax working with binary data as we have seen with the `<<>>` special form provided for working with _bitstrings_.
+
+## Binary data
+
+The binary type is a specialization on the bitstring type. Where bitstrings could be of any length (any number of [bits][wiki-bit]), binaries are where the number of [bits][wiki-bit] can be evenly divided by 8. That is, when working with binaries, we often think of things in terms of [bytes][wiki-bytes] (8 [bits][wiki-bit]). A [byte][wiki-bytes] can represent integer numbers from `0 - 255`. It is common to work with [bytes][wiki-byte] values in [hexadecimal][wiki-hexadecimal], `0x00 - 0xFF`.
+
+Binaries may use the bitstring special form `<<>>` to define a literal. When defining a literal, we can use integer literals (numbers greater than 255 will overflow and only the last 8 bits of the integer will be used) and string literals. By default, the `::binary` modifier is applied to the value. We can concatenate binaries with the `<>/2` operator.
```

Comment:
Two phrasing suggestions here:

> may use ... <<>>

Is there a different way to write them? As strings with `""`, right? To prevent students from using themselves this question, maybe say: "Binary literals are defined using..."?

> (numbers greater than 255 will overflow and only the last 8 bits of the integer will be used)

I feel like this statement is a bit too complicated to be in a parenthesis. I would make it a stand alone sentence so that it really sinks in and add a code sample that shows this behavior, e.g.
```elixir

<< 256 >> == << 0 >>
# => true
```